edb/server/metrics.py

```diff
@@ -194,6 +194,12 @@
     labels=('tenant', 'branch'),
 )
 
+extension_used = registry.new_labeled_gauge(
+    'extension_used',
+    'How many branches an extension is used by.',
```

Comment:
we should follow https://prometheus.io/docs/practices/naming/#metric-names whenever possible - something like `extension_used_branch_count`  means you can tell what 4 vs 5 means without having to refer back to the help text (which appears in the `/metrics` output, but isn't always propagated from there to a database like Prometheus, a visualizer like Grafana, etc)

Reply:
Sure. How about `extension_used_branch_count_current`, since our prometheus implementation in python seems to think we shouldn't end a metric with `_count`.